stories/form/form-textarea/doc-elements.tsx

```diff
@@ -0,0 +1,19 @@
+import { Text } from "../../../src/text/text";
+import styled from "styled-components";
+import React from "react";
+
+export const renderCustomCounter = (
+    maxLength: number,
+    currentValueLength: number
+): JSX.Element => {
+    const CustomCounterLabel = styled(Text.H6)`
```

Comment:
Do this outside of the function. Rendering inside the function is not recommended. 
https://styled-components.com/docs/faqs#why-should-i-avoid-declaring-styled-components-in-the-render-method